wye/urls.py

```diff
@@ -15,6 +15,10 @@
         name="dashboard"),
     url(r'^about/$', TemplateView.as_view(template_name='about.html',),
         name='about'),
+    url(r'^contact/$', ContactFormView.as_view(),
```

Comment:
Can we add tab in base.html with name "contact us"? On click on Contact us it should open /contact/ url defined in urls.py